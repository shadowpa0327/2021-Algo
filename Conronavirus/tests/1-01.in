42353747 -296476222 10
90534281
21742640
7999797
41016632
91959605
87534323
42139900
79498597
73806666
26302076
