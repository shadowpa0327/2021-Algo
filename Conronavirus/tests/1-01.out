629197255
773536032
561873496
927619112
152663839
519913717
298530163
112857566
851576255
96287611
